phpBB/includes/functions.php

```diff
@@ -4806,6 +4806,16 @@ function phpbb_get_avatar($row, $alt, $ignore_config = false)
 			'alt="' . ((!empty($user->lang[$alt])) ? $user->lang[$alt] : $alt) . '" />';
 	}
 
+	/**
+	* Event to modify HTML <img> tag of avatar
+	*
+	* @event core.get_avatar_after
+	* @var	string	html		The HTML <img> tag of generated avatar
+	* @since 3.1.6-RC1
+	*/
+	$vars = array('html');
```

Comment:
These event should have more (context) variables to allow for extensions to choose what to do.  For example, information about the user and the unprocessed avatar data, at a minimum, should be provided, just in case an extension might need them.  Creating this event just with html as the only exposed variable seems too limiting.
Even data that is available through html, if needed, would require parsing of the provided html code.
I would recommend adding extra variables as parameters to the event.